components/script/dom/node.rs

```diff
@@ -573,6 +579,29 @@ impl Node {
         window_from_node(self).client_rect_query(self.to_trusted_node_address())
     }
 
+    // https://drafts.csswg.org/cssom-view/#extension-to-the-element-interface
```

Comment:
This looks like it ought to be https://drafts.csswg.org/cssom-view/#dom-element-scrollwidth and https://drafts.csswg.org/cssom-view/#dom-element-scrollheight.